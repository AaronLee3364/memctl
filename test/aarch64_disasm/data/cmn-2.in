fffffff006c717d4 2b09011f
