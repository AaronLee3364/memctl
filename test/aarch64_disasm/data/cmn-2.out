fffffff006c717d4  CMN     W8, W9
